path_test.go

```diff
@@ -18,3 +18,86 @@ func TestParsePath(t *testing.T) {
 		log.Printf("di: %+v, di.M: %+v", di, di.M)
 	}
 }
+
+type PathTest struct {
+	Description string
+	Svg     string
+	Kinds   []InstructionType
+	XCoords []float64
+	YCoords []float64
+}
+
+var tests = []PathTest{
+	{
+		"relative h-line test",
+		`<svg viewBox="0 0 100 100"><path d="M0.000 0.000 h100.000 50.000" fill="#000000" stroke="#000000" stroke-width="2"/></svg>`,
+		[]InstructionType{MoveInstruction, LineInstruction, LineInstruction, PaintInstruction},
+		[]float64{0, 100, 150, 0},
+		[]float64{0, 0, 0, 0},
+	},
+	{
+		"absolute h-line test",
+		`<svg viewBox="0 0 100 100"><path d="M0.000 0.000 H100.000 50.000" fill="#000000" stroke="#000000" stroke-width="2"/></svg>`,
+		[]InstructionType{MoveInstruction, LineInstruction, LineInstruction, PaintInstruction},
+		[]float64{0, 100, 50, 0},
+		[]float64{0, 0, 0, 0},
+	},
+	{
+		"relative v-line test",
+		`<svg viewBox="0 0 100 100"><path d="M0.000 0.000 v100.000 50.000" fill="#000000" stroke="#000000" stroke-width="2"/></svg>`,
+		[]InstructionType{MoveInstruction, LineInstruction, LineInstruction, PaintInstruction},
+		[]float64{0, 0, 0, 0},
+		[]float64{0, 100, 150, 0},
+	},
+	{
+		"absolute v-line test",
+		`<svg viewBox="0 0 100 100"><path d="M0.000 0.000 V100.000 50.000" fill="#000000" stroke="#000000" stroke-width="2"/></svg>`,
+		[]InstructionType{MoveInstruction, LineInstruction, LineInstruction, PaintInstruction},
+		[]float64{0, 0, 0, 0},
+		[]float64{0, 100, 50, 0},
+	},
+}
+
+func TestParsePathList(t *testing.T) {
+	for _, test := range tests {
+		svg, err := ParseSvg(test.Svg, "test", 0)
+		require.NoError(t, err)
+
+		dis, _ := svg.ParseDrawingInstructions()
+		strux := []*DrawingInstruction{}
+		for di := range dis {
+			strux = append(strux, di)
+			log.Printf("di: %+v, di.M: %+v", di, di.M)
```

Comment:
Let's keep the tests as quiet as possible and use t.Logf here instead of log.Printf.